students/detra.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Detra</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <style>
+        body {
```

Comment:
💭 Would love to see this with a CSS style sheet!

Reply:
Oh sure thing. I wasn't really sure I could but I definitely will do that and add more css. Thx

Reply:
You don't have to update it with CSS 😊. The assignment doesn't ask for a stylesheet. But, if you could link your html to the main html, that would be lovely! 🤗